### ISSUE TITLE
分享的数据文件已失效

### ISSUE THREAD

**Issue by ifushu** [Ifjmww/FaultSeg3D_pytorch#2]
作者你好，很感谢你对于该工作的复现与分享。但是您分享的百度网盘链接已失效。是否能重新分享，或者发送到邮箱ifushu@163.com呢。万分感谢

**Comment by rangelgjc** [comment]
Yeah it is not working the link. Can you share it again and in other platforms
